Expand on Snowfork content on Bridge Hub page (#6167)
w3f/polkadot-wiki: docs/learn/learn-bridge-hub.md

id: learn-bridge-hub
title: Bridge Hub
sidebar_label: Bridge Hub
description: Overview of the Bridge Hub System Parachain.
keywords: [Bridge, XCM, Bridge Hub]
slug: ../learn-bridge-hub
---

The primary functionality of the {{ polkadot: Polkadot :polkadot }}{{ kusama: Kusama :kusama }}
relay chain is to secure the parachains and facilitate secure communication between them. All other
functionalities like asset transfers, governance, identities and bridging (a potentially resource

> intensive task) can benefit from operating separately on system parachains. System parachains are

Reviewer: ```suggestion
intensive task) can benefit from operating separately on [system parachains](./learn-system-chains.md). System parachains are
```

responsible for delegating functionality away from the relay chain for peformance reasons, taking

> advantage of the inherent parallelization the relay chain provides.

Reviewer: ```suggestion
advantage of the inherent parallelization the architecture of Polkadot provides.
```

The Bridge Hub system parachain operates on
{{ polkadot: Polkadot :polkadot }}{{ kusama: Kusama :kusama }}, and is responsible for faciliating

> bridges to the wider Web3 space. It contains all required bridge

Reviewer: ```suggestion
bridges to the wider Web3 space. It contains the required bridge
```

[pallets](../general/glossary.md#pallet) in its runtime, which enable trustless bridging with other
blockchain networks like {{ polkadot: Kusama :polkadot }}{{ kusama: Polkadot :kusama }}, Ethereum

> etc. The Bridge Hub uses the native token of the relay chain,

Reviewer: ```suggestion
blockchain networks like {{ polkadot: Kusama :polkadot }}{{ kusama: Polkadot :kusama }} and Ethereum
. The Bridge Hub uses the native token of the relay chain,
```

source and a target chain. Any bridge operating on the Bridge Hub will have on-chain (pallets) and
offchain (relayers) components.

> Examples of trustless on Bridge Hub include:

Reviewer 2: ```suggestion
Examples of trustless bridges on Bridge Hub include:
```

> - Snowfork, a decentralized bridge between Polkadot and Ethereum

Reviewer 2: should we specify that this is an Ethereum light client on Polkadot that syncs to Ethereum and keeps the Ethereum state up-to-date on Polkadot?

Author: Maybe this is good for the next PR, we can add those details in the followup 👍
